packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1120,17 +1120,7 @@ test.serial('writeGranulesFromMessage() on re-write saves granule records to Dyn
     expectedGranule
   );
   t.deepEqual(dynamoRecord, expectedGranule);
-  // ES fails here because of bad write logic in <TICKET>
```

Comment:
some of my changes seem to have resolved this test. I don't have a ticket number to refer to, so I'm not sure the exact issue that was happening. If I had to guess, I think the translating back to `limitedUpdateApiGranuleRecord` from the limited PG write could have resolved something, but I can't be certain. If anyone else knows what was going on with this test, that may be helpful.

Reply:
Looking into this - I'm uncomfortable that we don't know why this changed.

Reply:
Ok - I believe I know the answer, but cannot find the ticket - the *es updates* don't have a mutable field check for runinng/queued.      By forcibly writing the entire translated/written record in those cases in this PR, the nullable field logic is effectively set only in the postgres granule.   

You may have also addressed CUMULUS-3113 - if so, we should take the win on that ticket.